frontend/README.md

```diff
@@ -48,7 +48,7 @@ curl -X GET "https://localhost:8443/subscriptions/YOUR_SUBSCRIPTION_ID/resourceG
 
 Create or Update a HcpOpenShiftClusterResource
 ```bash
-curl -X PUT "https://localhost:8443/subscriptions/YOUR_SUBSCRIPTION_ID/resourceGroups/YOUR_RESOURCE_GROUP_NAME/providers/Microsoft.RedHatOpenshift/hcpOpenShiftClusters/YOUR_CLUSTER_NAME?api-version=2024-06-10-preview" -H "Content-Type: application/json" -d @mycluster.yaml
```

Comment:
is `mycluster.json` an existing file?

Reply:
No it's just an example.  I mainly wanted to fix the `.yaml` suffix as it was misleading.